snapcraft/internal/meta.py

```diff
@@ -201,7 +201,7 @@ def write_snap_directory(self):
 
                     file_utils.link_or_copy(source, destination)
 
-        self._record_snapcraft()
+        self._record_manifest()
```

Comment:
I like the renaming. Actually makes it easier to ack the code for manifest and find relevant test cases as well.